api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -611,6 +611,39 @@ type FelixConfigurationSpec struct {
 	// [Default: Auto]
 	BPFConntrackCleanupMode *BPFConntrackMode `json:"bpfConntrackMode,omitempty" validate:"omitempty,oneof=Auto Userspace BPFProgram"`
 
+	// BPFConntrackTimers overides the default values for the specified conntrack timer if
+	// set. It is a key-value make, where each value can be either a duration or a name of
+	// a Linux conntrack timeout to use.
+	//
+	// Possible values for the keys are: CreationGracePeriod, TCPPreEstablished,
+	// TCPEstablished, TCPFinsSeen, TCPResetSeen, UDPLastSeen, GenericIPLastSeen,
+	// ICMPLastSeen.
+	//
+	// Example:
+	//
+	// CreationGracePeriod: 15s
+	// TCPPreEstablished: nf_conntrack_tcp_timeout_syn_sent
+	// TCPFinsSeen: nf_conntrack_tcp_timeout_time_wait
+	//
+	// This would override 3 timers, one with 15 seconds and two with respective values
+	// taken from /proc/sys/net/netfilter/<name> files.
+	//
+	// Unset or incorrect values are replaced by the default values with a warning log for
+	// incorrect values.
+	//
+	// [Default:
+	//	CreationGracePeriod: 10s
+	//	TCPPreEstablished:   20s
+	//	TCPEstablished:      1h
+	//	TCPFinsSeen:         nf_conntrack_tcp_timeout_time_wait
```

Comment:
Not sure about allowing arbitrary sysctl lookups; feels a bit too powerful (and could allow user to trick felix into reading something that we don't want)?  Is there an obvious mapping from sysctls to our values so that we could just do `auto` and that means "use sysctl from the look-up-table"?

Failing that, we should at least have an explicit allow-list of sysctls.

Reply:
yes, `auto` sounds good, I can hardcode the auto values :+1: